odata-data-aggregation-ext/3.3 Transformations Producing a Subset.md

```diff
@@ -226,7 +226,8 @@ results in
 
 The `skip` transformation takes a non-negative integer $c$ as argument. Let $A$ be a copy of the input set with a total order that extends any existing order of the input set but is otherwise chosen by the service. The total order MUST be stable across requests.
 
-The transformation excludes from the output set the first $c$ occurrences in $A$. It keeps all remaining instances in the same order as they occur in $A$.
+If $A$ contains $c$ or fewer instances, the output set is empty.
+Otherwise, the transformation excludes from the output set the first $c$ occurrences in $A$. It keeps all remaining instances in the same order as they occur in $A$.
```

Comment:
Do you really want to change Data Aggregation? We should only do this in a working draft for CSD05.